doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -1,66 +1,165 @@
 .. _proofhandling:
 
--------------------
- Proof handling
--------------------
+----------
+Proof mode
+----------
 
-In Coq’s proof editing mode all top-level commands documented in
-Chapter :ref:`vernacularcommands` remain available and the user has access to specialized
-commands dealing with proof development pragmas documented in this
-section. They can also use some other specialized commands called
-*tactics*. They are the very tools allowing the user to deal with
-logical reasoning. They are documented in Chapter :ref:`tactics`.
+:gdef:`Proof mode <proof mode>` is used to prove theorems.
+Coq enters proof mode when you begin a proof,
+such as with the :cmd:`Theorem` command.  It exits proof mode when
+you complete a proof, such as with the :cmd:`Qed` command.  Tactics,
+which are available only in proof mode, incrementally transform incomplete
+proofs to eventually generate a complete proof.
 
-Coq user interfaces usually have a way of marking whether the user has
-switched to proof editing mode. For instance, in coqtop the prompt ``Coq <``   is changed into
-:n:`@ident <`   where :token:`ident` is the declared name of the theorem currently edited.
+When you run Coq interactively, such as through CoqIDE, Proof General or
+coqtop, Coq shows the current proof state--the incomplete proof--as you
```

Comment:
```suggestion
coqtop, Coq shows the current proof state—the incomplete proof—as you
```

Reply:
Ah, a subtle point.  The typewritten form of an "em dash" is `--`, while a hyphen is `-`.  The em dash is used in this case to introduce parenthetical information, while the hyphen is used to form compound words.  "em" as in the width of the letter "m".  In Unicode the en dash is  `—`.

https://www.grammarly.com/blog/hyphens-and-dashes/#:~:text=What's%20the%20Difference%20Between%20Dashes%20and%20Hyphens%3F&text=A%20hyphen%20(%2D)%20is%20a,a%20range%20or%20a%20pause.

I'll use parentheses instead.

Reply:
What I put here was precisely the unicode character for em dash.

Reply:
Doh!  Right you are.  Easy to confuse the two.